src/components/Button/native/Button.js

```diff
@@ -58,19 +43,19 @@ const Button = styled.TouchableOpacity`
       padding: ${p.theme.px([2, 3])};
     `,
   }[p.size])}
-  ${(p) => p.block ? 'width: 100%;' : ''}
-  ${(p) => p.disabled ? `
-    opacity: 0.5;
-  ` : ''}
+  ${(p) => p.block ? 'align-self: stretch;' : 'align-self: flex-start;'}
+  ${(p) => p.disabled ? 'opacity: 0.5;' : ''}
   `;
 
 Button.propTypes = {
+  ...TouchableOpacity.propTypes,
   color: PropTypes.string,
   kind: PropTypes.oneOf(['solid', 'outlined', 'transparent']),
-  shape: PropTypes.oneOf(['default', 'flat', 'rounded', 'circular']),
+  shape: PropTypes.oneOf(['default', 'flat', 'rounded']),
```

Comment:
Why was circular removed? Is it not part of the guideline anymore?

Reply:
Not part of the native app guideline

Reply:
We have just one spec across the system, the native and web components is just an implementation of the spec.
If the circular shape is removed from the spec, then remove the circular implementation for the web Button component too.